apps/site/src/app/apply/sections/Form/ProfileInformation.tsx

```diff
@@ -0,0 +1,55 @@
+import RequiredAsterisk from "../Components/RequiredAsterisk";
+import TextInput from "../Components/TextInput";
+import Textfield from "../Components/Textfield";
+import styles from "./Form.module.scss";
+
+export default function ProfileInformation() {
+	return (
+		<div className="flex flex-col gap-5 w-11/12">
+			<p className="text-4xl m-0 font-bold text-center max-[700px]:text-3xl">
+				Profile Information
+			</p>
+
+			<div className="flex gap-5 w-full max-[1000px]:flex-col max-[1000px]:items-center">
+				<TextInput
+					name="linkedin"
+					labelClass={styles.label}
+					labelText="LinkedIn"
+					inputClass={styles.input}
+					containerClass="flex flex-col w-6/12 max-[1000px]:w-full"
+					isRequired={false}
+					type="url"
+					placeholder="http://"
```

Comment:
Suggestion [repeating]: if we are using a placeholder, we should suggest https instead of plain http.